app/build.gradle

```diff
@@ -45,6 +45,8 @@ android {
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
 
         buildConfigField "String", "CONFERENCE_TIMEZONE", "\"Asia/Tokyo\""
+        buildConfigField "String", "API_ROOT", "\"https://droidkaigi.github.io/2017/\""
```

Comment:
Why did I divide `API_ROOT` and `STATIC_ROOT` though both are similar?
Because retrofit base url have to contain `/` on the end, and the static root url may be changed in the future.